<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDialog</class>
 <widget class="QDialog" name="MessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_msgbox" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>978</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_msgbox{background-color: rgba(0, 0, 0, 0.5);}</string>
   </property>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>316</y>
      <width>552</width>
      <height>345</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#centralWidget{image: url(:/3实时登机/Images/3实时登机/弹窗/小弹窗BG.png);}
</string>
    </property>
    <widget class="QLabel" name="label_txt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>552</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>是 否 确 认 ？</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_YES">
     <property name="geometry">
      <rect>
       <x>322</x>
       <y>240</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/2系统首页/Images/2系统首页/弹窗/确--认-拷贝.png);}
QPushButton:pressed{border-image: url(:/2系统首页/Images/2系统首页/弹窗/确--认-拷贝-2.png);}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_X">
     <property name="geometry">
      <rect>
       <x>485</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/2系统首页/Images/2系统首页/弹窗/关--闭-拷贝.png);}
QPushButton:pressed{border-image: url(:/2系统首页/Images/2系统首页/弹窗/关--闭-拷贝-按下.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>29</width>
       <height>29</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_NO">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border:0;image: url(:/2系统首页/Images/2系统首页/弹窗/取消-拷贝.png);}
QPushButton:pressed{image: url(:/2系统首页/Images/2系统首页/弹窗/取消-拷贝-2.png);}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>0</y>
       <width>400</width>
       <height>62</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 228, 255);</string>
     </property>
     <property name="text">
      <string>标题</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="flowTableWidget">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>100</y>
       <width>440</width>
       <height>210</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <zorder>label_txt</zorder>
    <zorder>pushButton_YES</zorder>
    <zorder>pushButton_NO</zorder>
    <zorder>label_title</zorder>
    <zorder>pushButton_X</zorder>
    <zorder>flowTableWidget</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
